base/kra/shared/webapps/kra/agent/kra/GrantRecovery.html

```diff
@@ -32,7 +32,7 @@
     <td  valign="top" align="right">
       <font size="-1" face="PrimaSans BT, Verdana, sans-serif">Recovery authorization reference number:<br></font>
     </td>
-    <td><INPUT TYPE="TEXT" NAME="recoveryID" SIZE=10 MAXLENGTH=99"></td>
+    <td><INPUT TYPE="NUMBER" NAME="recoveryID" MIN=0 MAX=99"></td>
```

Comment:
`min` and `max` on the number field dictate the maximum allowed recovery ID value. I think `99` is a little small. I'd perhaps just remove the `max` attribute. (Previously, `size=10` made the `input` element 10 characters wide, but allowed a number up to `maxlength=99` characters. `99 < (10^100 - 1)`). :-)

Reply:
Fixed :)

Reply:
Brilliant, thanks Pritam!